sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/implementation/JobRouterAdministrationsImpl.java

```diff
@@ -69,7 +69,7 @@ public final class JobRouterAdministrationsImpl {
     @ServiceInterface(name = "AzureCommunicationSe")
     public interface JobRouterAdministrationsService {
         @Patch("/routing/classificationPolicies/{id}")
-        @ExpectedResponses({200})
+        @ExpectedResponses({200,201})
```

Comment:
@cparisineti Is this expected?

Reply:
I see both 200, 201 in the service code  - https://skype.visualstudio.com/SCC/_git/ic3_spool_ccaap?path=/src/Router.Api/V2022_07_18_preview/Controllers/ClassificationPoliciesController.cs 

but the swagger indeed only shows only 200 - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/communication/data-plane/JobRouter/preview/2022-07-18-preview/communicationservicejobrouter.json